### PR TITLE
1.40 update notification

### PR DESCRIPTION
**Pull request by WorldLanguages**
image

(Technically the mp4 update was 1.39.2 but I think it still deserves attention)

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -248,8 +248,8 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_39": {
-    "message": "New addon: \"delete other costumes or sounds\", which adds an option to the right click context menu for costumes and sounds to delete all items except for the selected one. Go to $1 to enable it.",
+  "extensionUpdateInfo1_v1_40": {
+    "message": "New addon: \"delete sprites without confirmation\", which removes the new confirmation prompt that asks if you are sure you want to delete a project sprite. Go to $1 to enable it.",
```
[the remark this answers is not in the document]
Comment (WorldLanguages) [reply _locales/en/messages.json:252]: I would think everyone is aware of the new confirmation prompt. If they think the Scratch native one is added by us, it doesn't matter, we are still telling them how to disable it.

Comment (mxmou) [inline _locales/en/messages.json:252]:
```suggestion
    "message": "New addon: \"delete sprites without confirmation\", which removes the new confirmation prompt that asks if you are sure you want to delete a sprite. Go to $1 to enable it.",
```

File: _locales/en/messages.json [hunk 2]
```diff
@@ -258,8 +258,8 @@
   "sendFeedbackNotification": {
     "message": "send feedback"
   },
-  "extensionUpdateInfo2_v1_39": {
-    "message": "Other new addons added this update are \"asset conflict dialog\", \"customizable code editor grid\", \"customizable stage monitors\", and  \"remaining reply counter\"."
+  "extensionUpdateInfo2_v1_40": {
+    "message": "The \"save blocks as image\" addon now allows copying the images directly to the clipboard. Also, the \"project video recorder\" now exports mp4 files when supported by the browser."
```
Comment (mxmou) [inline _locales/en/messages.json:262]:
```suggestion
    "message": "The \"save blocks as image\" addon now allows copying the images directly to the clipboard. Also, the \"project video recorder\" now exports mp4 files if supported by the browser."
```

File: _locales/en/messages.json (hunk 1 above)
Comment (WorldLanguages) [reply _locales/en/messages.json:252]: The word "project" can also be removed in the PR

**Pull request merged**